cli/src/test/scala/com/ibm/sparktc/sparkbench/OutputTest.scala

```diff
@@ -19,18 +19,13 @@ class OutputTest extends FlatSpec with Matchers with BeforeAndAfterAll with Capt
   }
 
   "Specifying Console output" should "work" in {
-    val (out) = captureOutput(CLIKickoff.main(
-      Array("bin/spark-bench.sh", "workload", "kmeans", "-i", dataStuff.kmeansFile, "-o", "console", "-n", "5", "-k", "2"))
-    )
+    val (out) = captureOutput(CLIKickoff.main(Array(getClass.getResource("/etc/testConfFile3.conf").getPath)))
```

Comment:
I can answer that. It was like that I was using a method from Capturing that output a tuple of stuff, and then I switched to captureOutput which only captures stdout.